tests/study_tests/test_study.py

```diff
@@ -9,7 +10,7 @@
 import threading
 import time
 from typing import Any
-from typing import Callable
+from typing import Callable as typing_Callable
```

Comment:
How about defining the name as the camel case like this?
```suggestion
from typing import Callable as TypingCallable
```